StackOverflow/Model/DataDelegate.h

```diff
@@ -0,0 +1,16 @@
+//
+//  DataDelegate.h
+//  StackOverflow
+//
+//  Created by Malea on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol DataDelegate <NSObject>
```

Comment:
Once again naming, DataDelegate is so generic it does not tell me anything. What sort of data is this ? Is it questions, is it answers what ? Name things to clearly convey their role.